test/test_ops.py

```diff
@@ -422,12 +458,15 @@ def test_boxes_shape(self):
 
 
 class TestMultiScaleRoIAlign:
+    def make_obj(self, fmap_names=["0"], output_size=(7, 7), sampling_ratio=2, **kwargs):
```

Comment:
We need to avoid any mutable argument in methods. Here is why:

```python
>>> def x(v, d=[]):
...     d.append(v)
...     print(d)
... 
>>> 
>>> x(1, [])  # that works as expected because we pass a new list
[1]
>>> x(1, [])  # same here
[1]
>>> x(1)  # This addition happen on the single default list
[1]
>>> x(1) # oops!
[1, 1]
>>> x(1)
[1, 1, 1]
```